boards/common/esp32/Makefile.include

```diff
@@ -1,7 +1,6 @@
 # configure the serial interface
 PORT_LINUX ?= /dev/ttyUSB0
 PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.SLAB_USBtoUART*)))
-include $(RIOTMAKE)/tools/serial.inc.mk
 
 # reset tool configuration
 RESET ?= $(RIOTTOOLS)/esptool/espreset.py
```

Comment:
Here `PROG_DEV` is used just below but it would not be yet set.

Reply:
I opened #15531 to fix this problem.